src/Entity/Translation/MachineTranslation.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace App\Entity\Translation;
+
+use App\Utils\TimeUtils;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+use Exception;
+
+abstract class MachineTranslation
+{
+  /**
+   * @ORM\Id
+   * @ORM\GeneratedValue
+   * @ORM\Column(type="integer")
+   */
+  protected ?int $id = null;
+
+  /**
+   * @ORM\Column(type="string", length=5)
+   */
+  protected string $source_language;
+
+  /**
+   * @ORM\Column(type="string", length=5)
+   */
+  protected string $target_language;
+
+  /**
+   * @ORM\Column(type="string")
+   */
+  protected string $provider;
+
+  /**
+   * @ORM\Column(type="integer")
+   */
+  protected int $usage_count;
+
+  /**
+   * @ORM\Column(type="float")
+   */
+  protected float $usage_per_month;
```

Comment:
this number can be calculated in code and shown on the admin dashboard (next pr), but that way, it seems very hard to enable sorting using this column. So I included it in the database